diff -urN kdeedu-3.5.5.org/kig/configure.in.in kdeedu-3.5.5/kig/configure.in.in
--- kdeedu-3.5.5.org/kig/configure.in.in	2006-10-01 19:24:06.000000000 +0200
+++ kdeedu-3.5.5/kig/configure.in.in	2007-01-05 22:26:40.795502000 +0100
@@ -115,7 +115,7 @@
     __regina_py_save_ldflags="$LDFLAGS"
     __regina_py_save_libs="$LIBS"
     __regina_py_ok=0
-    for pyver in python python2.4 python2.3 python2.2; do
+    for pyver in python python2.5 python2.4 python2.3 python2.2; do
       for incdir in "/usr/include/$pyver" "/usr/local/include/$pyver" \
           "/usr/$pyver/include" "/usr/local/$pyver/include" \
           "$prefix/include/$pyver" "$prefix/$pyver/include"; do
@@ -140,9 +140,10 @@
             AC_MSG_RESULT([yes])
 	
             # Check for -lpython.
-            for pylib in "$pyver" python python2.3 python2.2; do
+            for pylib in "$pyver" python python2.5 python2.4 python2.3 python2.2; do
               for pylibdir in "/usr/lib" "/usr/local/lib" "/usr/lib/$pyver/config" \
-		"/usr/local/lib/$pyver/config"; do
+		"/usr/local/lib/$pyver/config" "/usr/lib64" "/usr/local/lib64" "/usr/lib64/$pyver/config" \
+		"/usr/local/lib64/$pyver/config" ; do
 		for extralibs in "" "-lpthread -lm -lutil -ldl"; do
                   AC_MSG_CHECKING([for $pylibdir/lib$pylib and $extralibs with $incdir/Python.h])
 		  LDFLAGS="$__regina_py_save_ldflags -L$pylibdir"
