--- kdeedu-050217/kstars/kstars/indi/webcam/PPort.cpp.orig	2005-02-11 22:44:47.000000000 +0100
+++ kdeedu-050217/kstars/kstars/indi/webcam/PPort.cpp	2005-02-20 15:38:28.641966104 +0100
@@ -16,7 +16,7 @@
     Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 
 */
-
+#ifdef __i386__
 #include "PPort.h"
 #include "port.h"
 
@@ -105,3 +105,4 @@
    TEST_VALIDITY;
    return (assignedBit[bit] == ID);
 }
+#endif
--- kdeedu-3.3.0/kstars/kstars/indi/webcam/port.cpp.orig	2004-07-17 17:52:59.000000000 +0000
+++ kdeedu-3.3.0/kstars/kstars/indi/webcam/port.cpp	2004-08-24 20:45:36.000000000 +0000
@@ -26,7 +26,7 @@
 // FreeBSD code by: Patrick Reynolds <reynolds@cs.duke.edu> and Charles
 //                  Henrich <henrich@msu.edu>
 
-
+#ifdef __i386__
 //#include "config.h"
 
 #include <stdio.h>
@@ -196,3 +196,4 @@
 #endif /* DEBUG */
 }
 #endif /* LOCKING */
+#endif
