diff -uNr kdeedu-3.0.4.orig/keduca/keduca/keduca.desktop kdeedu-3.0.4/keduca/keduca/keduca.desktop
--- kdeedu-3.0.4.orig/keduca/keduca/keduca.desktop	Sat Sep 28 20:28:28 2002
+++ kdeedu-3.0.4/keduca/keduca/keduca.desktop	Mon Nov 11 20:47:47 2002
@@ -43,7 +43,7 @@
 GenericName[zh_TW]=測試與考驗
 GenericName[zu]=Izivivinyo kanye Nokubhalwa kwezivivinyo
 Exec=keduca
-Icon=keduca
+Icon=keduca.png
 MimeType=application/x-edu
 Name=KEduca
 Name[af]=Keduca
diff -uNr kdeedu-3.0.4.orig/keduca/keduca/x-edu.desktop kdeedu-3.0.4/keduca/keduca/x-edu.desktop
--- kdeedu-3.0.4.orig/keduca/keduca/x-edu.desktop	Sat Sep 28 20:28:28 2002
+++ kdeedu-3.0.4/keduca/keduca/x-edu.desktop	Mon Nov 11 20:48:41 2002
@@ -43,7 +43,7 @@
 Comment[zh_TW]=Edu 文件
 Comment[zu]=Idokhumente ye Edu
 DefaultApp=keduca
-Icon=keduca
+Icon=keduca.png
 Type=MimeType
 MimeType=application/x-edu
 Patterns=*.edu;
diff -uNr kdeedu-3.0.4.orig/kvoctrain/kvoctrain/kvoctrain.desktop kdeedu-3.0.4/kvoctrain/kvoctrain/kvoctrain.desktop
--- kdeedu-3.0.4.orig/kvoctrain/kvoctrain/kvoctrain.desktop	Sat Sep 28 20:28:29 2002
+++ kdeedu-3.0.4/kvoctrain/kvoctrain/kvoctrain.desktop	Mon Nov 11 20:49:50 2002
@@ -53,7 +53,7 @@
 Name[zu]=KVocIsitimela
 MimeType=application/x-kvoctrain;
 Exec=kvoctrain
-Icon=kvoctrain
+Icon=kvoctrain.png
 DocPath=kvoctrain/index.html
 Type=Application
 Terminal=0
diff -uNr kdeedu-3.0.4.orig/kvoctrain/kvoctrain/x-kvtml.desktop kdeedu-3.0.4/kvoctrain/kvoctrain/x-kvtml.desktop
--- kdeedu-3.0.4.orig/kvoctrain/kvoctrain/x-kvtml.desktop	Sat Sep 28 20:28:29 2002
+++ kdeedu-3.0.4/kvoctrain/kvoctrain/x-kvtml.desktop	Mon Nov 11 20:49:48 2002
@@ -43,7 +43,7 @@
 Comment[zu]=Idokhumenti Yomlolongi Wokusetshenziswa kwamagama
 MimeType=application/x-kvoctrain
 DefaultApp=kvoctrain
-Icon=kvoctrain.xpm
+Icon=kvoctrain.png
 Type=MimeType
 Patterns=*.kvtml;
 MimeType=text/xml
