--- kdeedu-031204/kstars/kstars/indi/lx200driver.c.orig	2003-09-03 18:34:41.000000000 +0000
+++ kdeedu-031204/kstars/kstars/indi/lx200driver.c	2003-12-13 12:56:51.000000000 +0000
@@ -955,8 +955,8 @@
    ttyOptions.c_cflag &= ~PARENB;
 
    /* set baud rate */
-   ttyOptions.c_ispeed = B9600;
-   ttyOptions.c_ospeed = B9600;
+   cfsetispeed(&ttyOptions, B9600);
+   cfsetospeed(&ttyOptions, B9600);
 
   /* set input/output flags */
   ttyOptions.c_iflag = IGNBRK;
