diff -urN -x CVS kdeedu.orig/kstars/kstars/indi/README kdeedu/kstars/kstars/indi/README
--- kdeedu.orig/kstars/kstars/indi/README	Tue Feb  3 07:34:32 2004
+++ kdeedu/kstars/kstars/indi/README	Tue Feb  3 07:34:32 2004
@@ -100,4 +100,4 @@
 	more stuff
 
 ! For RCS Only -- Do Not Edit
-! @(#) $RCSfile$ $Date$ $Revision$ $Name$
+! @(#) $RCSfile$ $Date$ $Revision$ $Name$
diff -urN -x CVS kdeedu.orig/kstars/kstars/indi/eventloop.c kdeedu/kstars/kstars/indi/eventloop.c
--- kdeedu.orig/kstars/kstars/indi/eventloop.c	Mon Jun  7 07:59:20 2004
+++ kdeedu/kstars/kstars/indi/eventloop.c	Mon Jun  7 07:59:20 2004
@@ -421,4 +421,4 @@
 #endif
 
 /* For RCS Only -- Do Not Edit */
-static char *rcsid[2] = {(char *)rcsid, "@(#) $RCSfile$ $Date$ $Revision$ $Name$"};
+static char *rcsid[2] = {(char *)rcsid, "@(#) $RCSfile$ $Date$ $Revision$ $Name$"};
diff -urN -x CVS kdeedu.orig/kstars/kstars/indi/eventloop.h kdeedu/kstars/kstars/indi/eventloop.h
--- kdeedu.orig/kstars/kstars/indi/eventloop.h	Mon Jun  7 07:59:20 2004
+++ kdeedu/kstars/kstars/indi/eventloop.h	Mon Jun  7 07:59:20 2004
@@ -105,5 +105,5 @@
 #endif
 
 /* For RCS Only -- Do Not Edit
- * @(#) $RCSfile$ $Date$ $Revision$ $Name$
+ * @(#) $RCSfile$ $Date$ $Revision$ $Name$
  */
diff -urN -x CVS kdeedu.orig/kstars/kstars/indi/indidevapi.h kdeedu/kstars/kstars/indi/indidevapi.h
--- kdeedu.orig/kstars/kstars/indi/indidevapi.h	Mon Jun  7 07:59:20 2004
+++ kdeedu/kstars/kstars/indi/indidevapi.h	Mon Jun  7 07:59:20 2004
@@ -348,5 +348,5 @@
 #endif
 
 /* For RCS Only -- Do Not Edit
- * @(#) $RCSfile$ $Date$ $Revision$ $Name$
+ * @(#) $RCSfile$ $Date$ $Revision$ $Name$
  */
diff -urN -x CVS kdeedu.orig/kstars/kstars/indi/indiserver.c kdeedu/kstars/kstars/indi/indiserver.c
--- kdeedu.orig/kstars/kstars/indi/indiserver.c	Mon Jun  7 07:59:20 2004
+++ kdeedu/kstars/kstars/indi/indiserver.c	Mon Jun  7 07:59:20 2004
@@ -591,4 +591,4 @@
 }
 
 /* For RCS Only -- Do Not Edit */
-static char *rcsid[2] = {(char *)rcsid, "@(#) $RCSfile$ $Date$ $Revision$ $Name$"};
+static char *rcsid[2] = {(char *)rcsid, "@(#) $RCSfile$ $Date$ $Revision$ $Name$"};
diff -urN -x CVS kdeedu.orig/kstars/kstars/indi/lilxml.c kdeedu/kstars/kstars/indi/lilxml.c
--- kdeedu.orig/kstars/kstars/indi/lilxml.c	Mon Jun  7 07:59:20 2004
+++ kdeedu/kstars/kstars/indi/lilxml.c	Mon Jun  7 07:59:20 2004
@@ -742,4 +742,4 @@
 #endif
 
 /* For RCS Only -- Do Not Edit */
-static char *rcsid[2] = {(char *)rcsid, "@(#) $RCSfile$ $Date$ $Revision$ $Name$"};
+static char *rcsid[2] = {(char *)rcsid, "@(#) $RCSfile$ $Date$ $Revision$ $Name$"};
diff -urN -x CVS kdeedu.orig/kstars/kstars/indi/lilxml.h kdeedu/kstars/kstars/indi/lilxml.h
--- kdeedu.orig/kstars/kstars/indi/lilxml.h	Mon Jun  7 07:59:20 2004
+++ kdeedu/kstars/kstars/indi/lilxml.h	Mon Jun  7 07:59:20 2004
@@ -242,7 +242,7 @@
  */
 
 /* For RCS Only -- Do Not Edit
- * @(#) $RCSfile$ $Date$ $Revision$ $Name$
+ * @(#) $RCSfile$ $Date$ $Revision$ $Name$
  */
 
 #endif	/* LILXML_H */
diff -urN -x CVS kdeedu.orig/libkdeedu/extdate/extdatetime.cpp kdeedu/libkdeedu/extdate/extdatetime.cpp
--- kdeedu.orig/libkdeedu/extdate/extdatetime.cpp	Sun Jun  6 04:39:47 2004
+++ kdeedu/libkdeedu/extdate/extdatetime.cpp	Sat Aug 21 16:56:06 2004
@@ -280,6 +280,7 @@
 {
 	//JD 2451545 (01 Jan 2000) was a Saturday, which is dayOfWeek=6.
 	int a_day = (( jd() - 2451545 + 6 ) % 7);
+	if ( a_day < 0 ) a_day += 7;
 	return (a_day == 0) ? 7 : a_day;
 }
 
@@ -517,8 +518,11 @@
 ExtDate  ExtDate::addMonths( int months ) const
 {
 	int	a_month = month() + months;
-	int	a_year = year() + (a_month - 1)/12;	// month : [1..12]
-	a_month  = 1 + (a_month -1) % 12;
+	int	a_year = year() + (a_month - 1)/12;
+	while ( a_month < 1 ) {
+		 a_month += 12;
+		 a_year--;
+	}
 	return ExtDate(a_year, a_month, day());
 }
 
