diff -urN -x CVS kdeedu.orig/libkdeedu/extdate/extdatetime.cpp kdeedu/libkdeedu/extdate/extdatetime.cpp
--- kdeedu.orig/libkdeedu/extdate/extdatetime.cpp	Sun Jun  6 04:39:47 2004
+++ kdeedu/libkdeedu/extdate/extdatetime.cpp	Sat Aug 21 16:56:06 2004
@@ -280,6 +280,7 @@
 {
 	//JD 2451545 (01 Jan 2000) was a Saturday, which is dayOfWeek=6.
 	int a_day = (( jd() - 2451545 + 6 ) % 7);
+	if ( a_day < 0 ) a_day += 7;
 	return (a_day == 0) ? 7 : a_day;
 }
 
@@ -517,8 +518,11 @@
 ExtDate  ExtDate::addMonths( int months ) const
 {
 	int	a_month = month() + months;
-	int	a_year = year() + (a_month - 1)/12;	// month : [1..12]
-	a_month  = 1 + (a_month -1) % 12;
+	int	a_year = year() + (a_month - 1)/12;
+	while ( a_month < 1 ) {
+		 a_month += 12;
+		 a_year--;
+	}
 	return ExtDate(a_year, a_month, day());
 }
 
